dot/state/block_finalisation.go

```diff
@@ -254,7 +265,9 @@ func (bs *BlockState) handleFinalisedBlock(currentFinalizedHash common.Hash) err
 		// prune all the subchain hashes state tries from memory
```

Comment:
I think we can update the comment to `prune all the subchain hashes state tries from database`